x-pack/plugins/embeddable_enhanced/public/actions/panel_notifications_action.test.ts

```diff
@@ -91,16 +123,19 @@ describe('PanelNotificationsAction', () => {
       expect(result).toBe(false);
     });
 
-    test('returns true when in "edit" mode', async () => {
-      const context = createContext([{}], true);
-      const action = new PanelNotificationsAction();
+    test('returns true when switching to "edit" mode', async () => {
+      const context = createContext([{}]);
 
+      const action = new PanelNotificationsAction();
+      (context.embeddable as PublishesWritableViewMode).setViewMode(ViewMode.EDIT);
       const result = await action.isCompatible(context);
       expect(result).toBe(true);
     });
 
-    test('returns false when no embeddable has no events', async () => {
-      const context = createContext([], true);
+    test('returns false when in edit mode but embeddable has no events', async () => {
```

Comment:
Should the old test case of 'returns false when no embeddable has no events' be preserved?

Reply:
This is testing the same concept, I just renamed it to be more accurate to what it was actually testing - the old phrasing ("when no embeddable (?) has no events") was confusing to me 😆

Reply:
Thanks for clarifying. LGTM